tests/test_cli.py

```diff
@@ -844,6 +844,21 @@ def test_gc(runner):
         assert not os.path.exists(run1_dir)
 
 
+def test_sweep_pause(runner, live_mock_server, test_settings):
+    live_mock_server.set_ctx({"resume": True})
+    sweep_config = {
+        "name": "My Sweep",
+        "method": "grid",
+        "parameters": {"parameter1": {"values": [1, 2, 3]}},
+    }
+    sweep_id = wandb.sweep(sweep_config)
+    assert sweep_id == "test"
+    assert runner.invoke(cli.sweep, ["--pause", sweep_id]).exit_code == 0
+    assert runner.invoke(cli.sweep, ["--resume", sweep_id]).exit_code == 0
+    assert runner.invoke(cli.sweep, ["--stop", sweep_id]).exit_code == 0
+    assert runner.invoke(cli.sweep, ["--cancel", sweep_id]).exit_code == 0
```

Comment:
can we add checks here that the status was updated (call `get_sweep_state` and assert that state = `paused`, etc.)